SonarQube.Common/AnalysisConfig/AnalysisConfig.cs

```diff
@@ -28,14 +28,14 @@ public class AnalysisConfig
 
         #region SonarQube project properties
 
+        public string SonarQubeHostUrl { get; set; }
+
```

Comment:
Previously the code looked in the config file to find the path to the sonar-runner.properties file, then looked in the sonar-runner.properties file to find the SonarQube URL. The URL is now put directly into the config file. The post-processor doesn't need the username/password as it doesn't call the SonarQube server directly; it just uses the URL build the link to the dashboard to embed in the reports.